orbit-2-livedata/README.md

```diff
@@ -0,0 +1,47 @@
+# Orbit 2 Live Data plugin
+
+The Live Data plugin provides:
+
+- [Container](../orbit-2-core/src/main/java/com/babylon/orbit2/Container.kt)
+  extensions for LiveData streams.
+
+## Including the module
+
+```kotlin
+implementation("com.babylon.orbit2:orbit-livedata:<latest-version>")
+```
+
+## LiveData streams
+
+The Core DSL contains the following Container extension properties:
+
+- stateLiveData
+- sideEffectLiveData
+
+This is the recommended way to subscribe to a Container in Android.
```

Comment:
should all instances of the main objects/functions link to the relevant code?

For example, `Container` here, `stateLiveData` etc above?